src/backend/expungeservice/models/charge_types/duii.py

```diff
@@ -0,0 +1,25 @@
+from expungeservice.models.charge_types.charge import Charge
+from expungeservice.models.expungement_result import TypeEligibility, EligibilityStatus
+
+
+class Duii(Charge):
+
+    def _default_type_eligibility(self):
+
+        if not self.disposition:
+            return TypeEligibility(EligibilityStatus.NEEDS_MORE_ANALYSIS, reason='Further Analysis Needed - No disposition')
+
+        elif self.disposition.ruling[0:8].lower() == 'diverted':
+            return TypeEligibility(EligibilityStatus.INELIGIBLE, reason='Ineligible under 137.225(8)(b)')
+
+        elif self.acquitted():
+            return TypeEligibility(EligibilityStatus.NEEDS_MORE_ANALYSIS, reason='Further Analysis Needed')
+
+        else:
+            """
+            TODO: other dispositions than "convicted" are possible and are not handled here.
+            See issue: https://github.com/codeforpdx/recordexpungPDX/issues/601
+
+            TODO: reason can be expanded to "137.225(7)(a) - Traffic offenses are ineligible"
```

Comment:
The thinking behind this second todo: It would be great IMO to start providing more readable reason strings for most expungement results. This is a larger and separate issue though.